code/game/gamemodes/wizard/spellbook.dm

```diff
@@ -107,9 +149,13 @@
 	dat += "<h1>A Wizard's Catalogue Of Spells And Artifacts</h1><br>"
 	dat += "<h2>[uses] point\s remaining (<a href='?src=\ref[src];refund=1'>Get a refund</a>)</h2><br>"
 	dat += "<em>This book contains a list of many useful things that you'll need in your journey.</em><br>"
-	dat += "<strong>KNOWN SPELLS:</strong><br><br>"
+	dat += "<span style=\"color:blue\"><strong>KNOWN SPELLS:</strong></span><br><br>"
 
-	var/list/shown_spells = get_available_spells()
+	var/list/shown_spells = get_all_wizard_spells()
```

Comment:
Any particular reason you're copying those lists, rather than using them directly ?